Doc/library/argparse.rst

```diff
@@ -189,7 +189,8 @@ ArgumentParser objects
                           formatter_class=argparse.HelpFormatter, \
                           prefix_chars='-', fromfile_prefix_chars=None, \
                           argument_default=None, conflict_handler='error', \
-                          add_help=True, allow_abbrev=True, exit_on_error=True)
+                          add_help=True, allow_abbrev=True, exit_on_error=True,\
```

Comment:
Minor spacing issue.  Match the space before the backslash as done in the preceding lines.